.github/workflows/update-develop.yml

```diff
@@ -0,0 +1,38 @@
+name: Update develop
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  update-develop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 100
+      - name: Update develop
+        run: |
+          set -eu
```

Comment:
I recall reading that the default on GitHub Actions is `set -e -o pipefail`. So adding `set -u` could be useful. I'm not sure that it's worth the visualize noise of adding `set -u` to every `run` block, though it could be useful during development. I'm thinking more broadly here than just this one particular action.